docs/examples/tensorflow/tensorflow-dataset.ipynb

```diff
@@ -0,0 +1,426 @@
+{
```

Comment:
You need to rerun this notebook

Reply:
Yeah, I know. I just committed it as is, because I wanted to see what will happen in the Ci when it's run and if it will still be this low.

Reply:
I removed `std` value in `CropMirrorNormalize` by mistake, it's ok again.